src/test-support/mocks/src/meshes/_/dataplanes/_.ts

```diff
@@ -31,6 +31,9 @@ export default ({ fake }: EndpointDependencies): MockResponder => (req) => {
         }),
         inbound: [
           {
+            health: {
+              ready: fake.datatype.boolean(),
+            },
```

Comment:
Am I right in thinking that sometimes this isn't set? If so this should only randomly be set.

Reply:
I’m not sure. I _think_ `ready` can indeed be absent, but I’m not sure `health` can.

Reply:
Right yeah, we should make sure we provide mocks that replicate that

Reply:
I’ve updated the mocking of inbounds so they have the same source and randomized this part.